examples.py

```diff
@@ -0,0 +1,78 @@
+examplePersonAlive = {

```

Comment:
Imagine using camelCase in python which universally is written in snake_case

Reply:
Welp